OverloadedConnectionsPluginTest.test failure

*master*

Failed on Sonar build on master branch: https://hazelcast-l337.ci.cloudbees.com/view/Lab/job/Hazelcast-3.x-sonar-lab/44/testReport/junit/com.hazelcast.internal.diagnostics/OverloadedConnectionsPluginTest/test/

Stacktrace:
```
java.lang.AssertionError: '25-03-2018 21:31:03 OverloadedConnections[]
25-03-2018 21:31:03 OverloadedConnections[]
25-03-2018 21:31:04 OverloadedConnections[]
25-03-2018 21:31:04 OverloadedConnections[]
25-03-2018 21:31:04 OverloadedConnections[]
25-03-2018 21:31:04 OverloadedConnections[]
25-03-2018 21:31:04 OverloadedConnections[]
25-03-2018 21:31:05 OverloadedConnections[]
25-03-2018 21:31:05 OverloadedConnections[]
25-03-2018 21:31:05 OverloadedConnections[]
25-03-2018 21:31:05 OverloadedConnections[]
25-03-2018 21:31:05 OverloadedConnections[]
... much much more
```

According to our monitoring, the machine was not under any exceptional heavy load.

--- a/hazelcast/src/test/java/com/hazelcast/internal/diagnostics/OverloadedConnectionsPluginTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/diagnostics/OverloadedConnectionsPluginTest.java
@@ -25,6 +25,7 @@ import com.hazelcast.map.impl.operation.GetOperation;
 import com.hazelcast.nio.Packet;
 import com.hazelcast.spi.impl.operationservice.impl.DummyOperation;
 import com.hazelcast.spi.impl.operationservice.impl.operations.Backup;
+import com.hazelcast.spi.properties.GroupProperty;
 import com.hazelcast.test.AssertTask;
 import com.hazelcast.test.HazelcastSerialClassRunner;
 import com.hazelcast.test.annotation.SlowTest;
@@ -57,7 +58,8 @@ public class OverloadedConnectionsPluginTest extends AbstractDiagnosticsPluginTe
         Config config = new Config()
                 .setProperty(OverloadedConnectionsPlugin.PERIOD_SECONDS.getName(), "1")
                 .setProperty(OverloadedConnectionsPlugin.SAMPLES.getName(), "10")
-                .setProperty(OverloadedConnectionsPlugin.THRESHOLD.getName(), "10");
+                .setProperty(OverloadedConnectionsPlugin.THRESHOLD.getName(), "2")
+                .setProperty(GroupProperty.IO_OUTPUT_THREAD_COUNT.getName(), "1");
 
         local = Hazelcast.newHazelcastInstance(config);
         serializationService = getSerializationService(local);

